vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -291,6 +291,9 @@ and
                 do! setupProjectsAfterSolutionOpen() 
             }
         setupProjectsAfterSolutionOpen() |> Async.StartImmediate
+
+        let theme = package.ComponentModel.DefaultExportProvider.GetExport<ISetThemeColors>().Value
```

Comment:
Can Value ever be null?
Can GetExport<> ever be null?
These are C# interop APIs.   I imagine they will return null under some error conditions or other?